build.gradle

```diff
@@ -4,7 +4,7 @@ subprojects {
   apply plugin: "signing"
 
   group = "com.marklogic"
-  version = "1.3.0"
+  version = "1.3-SNAPSHOT"
```

Comment:
Shouldn't this be 1.3.0-SNAPSHOT if the eventual build will be 1.3.0 at release time?

I'm not sure we've setup real conventions on this project to say for certain.

Reply:
1.3-SNAPSHOT is the convention, so this is good.